Configuration/TCA/tx_typo3forum_domain_model_user_notification.php

```diff
@@ -11,7 +11,7 @@
 		'type' => 'type',
 		'crdate' => 'crdate',
 		'delete' => 'deleted',
-		'sortby' => 'crdate DESC',
+		'sortby' => 'ORDER BY crdate',
```

Comment:
Are you sure that `sortby` requires the entire `ORDER BY` statement? [The docs](https://docs.typo3.org/typo3cms/TCAReference/Reference/Ctrl/Index.html) are not especially clear on this, but it seems that just the field name is required. Or should this be `default_sortby`, also?